Entity.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Entity.h"
+#include <string>
+#include <iostream>
+
+Entity::Entity(const int health,const int attackDamage,const std::string name) : Name(name){
+    this->MaxHealth      =   health;
+    this->Health         =   this->MaxHealth;
+    this->AttackDamage   =   attackDamage;
+    this->isDead         =   false;
```

Comment:
Miért nem elég a Health értékét vizsgálni?

Reply:
@wajzy Arra készülünk, hogy a későbbiekben szükség lesz heal-elésre is, ezért mentjük el a maximum életerő pontot, és a maximum életerő azért nem const, mert arra számítunk, hogy azt is lehet majd növelni valamely módon. Amennyiben kéri, kivesszük ezt a featuret.